python/drned_xmnr/op/setup_op.py

```diff
@@ -41,6 +41,22 @@ def perform(self):
                 pass
         try:
             shutil.copytree(self.drned_skeleton, target)
+        except OSError as ose:
+            if ose.errno == errno.EEXIST:
+                msg = "The target {0} already exists. Did you use `overwrite' parameter?" \
+                      .format(target)
+            else:
+                msg = "Failed to copy the `drned-skeleton' directory."
+            raise ActionError(msg)
+        target = os.path.join(self.dev_test_dir, "drned")
+        if self.overwrite and os.path.exists(target):
+            try:
+                shutil.rmtree(target, ignore_errors=True)
+                os.remove(target)
+            except OSError:
+                pass
```

Comment:
I'm afraid ignoring this exception means that when `rmtree` fails and subsequent `copytree` fails too, the user would be confused by the message "The target ... already exists. Did you use overwrite parameter?".

Reply:
I interpreted the code as if rmtree fails, it is because the target did not exist, and hence the copytree will succeed.

Reply:
I see. My point was that the `OSError` might be not only `FileNotFoundError`, but also e.g. `PermissonError` in which case `copytree` would likely fail and the error message would be pretty confusing. Can you fix this older bug too?